common/src/Assets/EntityDefinition.cpp

```diff
@@ -69,18 +71,17 @@ namespace TrenchBroom {
         }
 
         size_t EntityDefinition::usageCount() const {
-            return m_usageCount;
+            return m_usageCount->load();
```

Comment:
In general, I prefer to use the operators provided by atomic instead of the member functions, e.g. `m_usageCount += 1` instead of `fetch_add(1)` etc. because the fact that we're hadling an atomic value doesn't matter here. You don't need to know that to understand the code IMO.

Reply:
I prefer making it as explicit as possible. Trust C++ to give us 2 choices to argue about.

If you insist I can change it.

Reply:
No, I don't insist.

Reply:
Oh, too late.